_pages/harmonize/mapping.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: inner
+title: Using Model-to-Model Mapping
+permalink: /harmonize/mapping/
+---
+
+Use a model-to-model mapping to quickly and easily define how to harmonize source data to produce entity instances, without writing any code. See the following topics for details:
+* [What is Model-to-Model Mapping?](#what-is-model-to-model-mapping?)
+* [Creating a Mapping](#creating-a-mapping)
+* [Changing the Mapping Source Document](#changing-the-mapping-source-document)
+* [Using a Mapping for Harmonization](#using-a-mapping-for-harmonization)
+* [Modifying an Existing Mapping](#modifying-an-existing-mapping)
+* [Understanding the Harmonization Code](#understanding-the-harmonization-code)
+* [Model-to-Model Mapping Limitations](#model-to-model-mapping-limitations)
+
+## What is Model-to-Model Mapping?
+
+The process of harmonization extracts values from your source data and uses them to create canonical instances of your entity model. A model-to-model mapping defines the source property from which to extract a given entity property value.
+
+For example, the following diagram illustrates a case where the **sku** entity property is extracted from the **SKU** source property and the **price** entity property is extracted from the **price** source property.
+
+![Basic Mapping Diagram]({{site.baseurl}}/images/mapping/m2m-basic.png)
+
+A model-to-model mapping enables you to encode such relationships so Data Hub Framework can act on them.
+
+The models in "model-to-model mapping" are your source model and your entity model. The source model is inferred from your source data, rather than explicitly defined in the same way as your entity model. When you create a mapping using the QuickStart **Mappings** view, QuickStart does this inferencing for you.

```

Comment:
"infers this for you"

Reply:
I'll fix it and add it to the pull req. And then roguishly merge it without your additional blessing.